content/en/docs/install-operate/installation.md

```diff
@@ -352,3 +352,15 @@ cosign copy docker.io/falcosecurity/falco:0.35.0 your-registry/falco:0.35.0
 ```
 
 And you'll be able to easily verify that the image in your registry was not tampered with!
+
+## Troubleshooting
+
+### falco-driver-loader: Unable to find a prebuilt falco eBPF probe / module
+
+This error explains that the falco driver loader tool, which loads the Falco driver, was not able to find a pre-built driver, whether as eBPF probe or kernel module, from the repository available at [download.falco.org](https://download.falco.org) - you can easily search the supported targets at [download.falco.org/driver/site](https://download.falco.org/driver/site/index.html).
+
+This means that there's no prebuilt driver available for the the kernel running on the machine where Falco is going to be installed.
+
+Anyway, you can add your kernel release version to the [build grid](https://github.com/falcosecurity/test-infra/tree/master/driverkit/config) the drivers are going to be built for. You can follow [this tutorial](tutorials/add-prebuilt-driver-config.md) to contribute the required configuration.
+
+> **Note** that there's a specific set of Linux distrbutions for which kernel are released, supported by the current pipeline to distribute prebuilt drivers. In detail, [driverkit](https://github.com/falcosecurity/driverkit) is the tool is leveraged under the hood, and that needs to support the specific Linux distribution. You can verify it [here](https://github.com/falcosecurity/driverkit/tree/master/pkg/driverbuilder/builder).
```

Comment:
I agree, but as of now we don't have a matrix like in markdown or something fancy in the repository or elsewhere.
We could embed a matrix directly in this documentation and leave a link to the source of truth, just in case of possible out of sync documentation. WDYT?